### PR TITLE
Add seperator attribute for url making

### PR DESCRIPTION
**Pull request by Shrhawk**
AssetLocator methods to_url and from_url for "relative_path" assets

### CODE REVIEW

**Comment by Shrhawk** [comment]
@symbolist @cpennington kindly review this

File: opaque_keys/edx/locator.py [hunk 1]
```diff
@@ -924,16 +927,17 @@ def map_into_course(self, course_key):
         """
         return self.replace(course_key=course_key)
 
-    def _to_string(self):
+    def _to_string(self, separator='@'):
```
Comment (symbolist) [inline opaque_keys/edx/locator.py]: Can we call this `block_id_seperator`? It would be a bit more descriptive.

**Comment by symbolist** [comment]
@Shrhawk can you also add test coverage for the new methods?

**Comment by adampalay** [comment]
@Shrhawk , what's the story with this PR?

**Comment by adampalay** [comment]
@Shrhawk , please update this PR?

**Comment by cpennington** [comment]
I think we decided we weren't doing this. I'm going to close it, and it can be reopened if needed.

**Pull request closed without merging**